github/ci/prow-deploy/files/orgs.yaml

```diff
@@ -813,6 +817,11 @@ orgs:
         privacy: closed
         repos:
           tekton-tasks-operator: write
+      test-benchmarks-maintainers:
+        maintainers:
+          - alicefr
+        repos:
+          test-benchmarks: admin
```

Comment:
@alicefr while I am ok with you having admin rights, I am not sure whether you will really need those?

Reply:
@dhiller I think I put from copy-pasting from the other. Let me know if I need to remove it. Probably, initially I'll be the only contributor for pushing the initial code and I need to automerge the PRs